romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -49,6 +49,38 @@ services:
     api:
       host: 0.0.0.0
       port: 9604
+      hooks:
```

Comment:
YAML supports comments. I think our YAML config file(s)should be generously commented. They should have a header and individual sections should have their own section.

Any open source solution I have seen uses thoroughly commented config files as a means to document their system and let the user know what options exist and how it's used.